webapp/src/components/ResourceMetadata.tsx

```diff
@@ -52,15 +97,34 @@ export default class ResourceMetadata extends React.Component<IPropTypes, never>
                       <p className ="info-edit" onClick={this.props.promptEditPrivacy}>edit</p>
                     </div>
                 </div>
+                <div className="info-group">
+                    <span className="info-header">Getting started</span>
+                    <p>
+                      <a className="info-link" href="https://www.hydroshare.org/">Starter notebook</a>
+                    </p>
+                </div>
+                <BigToolTip 
+                        title={archiveText}>
+                  <button className="archive-resource"
+                    onClick={showConfirmArchiveResourceModal}>
+                    <span>Archive resource</span>
+                  </button>
+                </BigToolTip>
```

Comment:
As cool as this is, it increases our bundle size by 20%. Up to you whether or not this is worth it. If we do keep it, you'll need to add `@material-ui/core` to the dependencies though (I currently can't build it without running `yarn add @material-ui/core`).

Reply:
this can be removed since we're changing this to be 'remove from workspace" so we don't need an explanation anymore.